### DISCUSSION TITLE
How to implement callbacks for cell and row focus change?

### DISCUSSION THREAD

**Discussion by scarletjack** [Comcast/react-data-grid#2708] [Q&A]
Loving the grid so far guys, I have a question though.

Is there a way to implement callbacks based on the focused cell/row element? 

For example:
```js
let onCellFocusChange = (e) =>{
      console.log(e) // e being an object containing the data/index/id of the current row
}
```

I basically want to be able to set the selected row based on keyboard input. and setting active data as a state based on current selected row after. Is my approach correct? Are there alternatives?

**Comment by phanect** [comment]
I'm sending a PR to add `onCellSelected` event handler: #3328
Currently waiting for review and approval.
